organizer-guide.Rmd

```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
 
 ## CoFests! Goals {- .unlisted}
 
-- Build community
-- Develop high interest topics
-- Open science
+- Build an inclusive community that is supportive of newcomers
+- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
```

Comment:
```suggestion
- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science) for open collaboration. 
```
Could a main goal of Cofests be the collaborative experience? So maybe specifying one of the key aspects of open science could be necessary.

Reply:
Ah, I really like the of pointing out collaboration is a key aspect of open science! I'll slightly rephrase this but very grateful for the suggestion and stressing the importance of collaboration. Thanks!